wire/internal/wire/wire_test.go

```diff
@@ -453,25 +454,28 @@ type testCase struct {
 //
 func loadTestCase(root string, wireGoSrc []byte) (*testCase, error) {
 	name := filepath.Base(root)
-	pkg, err := ioutil.ReadFile(filepath.Join(root, "pkg"))
+	pkg, err := readFile(filepath.Join(root, "pkg"))
 	if err != nil {
 		return nil, fmt.Errorf("load test case %s: %v", name, err)
 	}
 	var wantProgramOutput []byte
 	var wantWireOutput []byte
-	wireErrb, err := ioutil.ReadFile(filepath.Join(root, "want", "wire_errs.txt"))
+	wireErrb, err := readFile(filepath.Join(root, "want", "wire_errs.txt"))
 	wantWireError := err == nil
 	var wantWireErrorStrings []string
 	if wantWireError {
 		wantWireErrorStrings = strings.Split(string(wireErrb), "\n\n")
+		for i, s := range wantWireErrorStrings {
+			wantWireErrorStrings[i] = strings.TrimSpace(s)
```

Comment:
I don't understand this addition. Shouldn't all the error strings already match?